ApiAuthTokenGenerator.Tests/V1/Gateways/AuthTokenDatabaseGatewayTests.cs

```diff
@@ -65,5 +66,62 @@ public void InsertedRecordShouldBeInTheDatabase()
             defaultRecordRetrieved.ApiEndpointNameLookupId.Should().Be(tokenRequest.ApiEndpoint);
             defaultRecordRetrieved.ApiLookupId.Should().Be(tokenRequest.ApiName);
         }
+
+        [Test]
+        public void TokenRecordShouldBeRetrievedIfValidIdIsSupplied()
+        {
+            var tokenDataInDb = AddTokebRecordToTheDatabase();
```

Comment:
```suggestion
            var tokenDataInDb = AddTokenRecordToTheDatabase();
```

Sorry, a little pedantic 😅

Reply:
Ok so my github skills are useless...(Goes to look how to do suggested changes)